rr_arduino/rr_arduino.ino

```diff
@@ -174,3 +181,29 @@ void get_battery_state(long current_time){
     prev_time = millis();
   }
 }
+
+void raw_to_odom(double vel, int str_angle) {
```

Comment:
try to follow the function naming convention in https://google.github.io/styleguide/cppguide.html#Function_Names